openhtf/output/web_gui/__init__.py

```diff
@@ -387,13 +387,53 @@ def on_unsubscribe(self):
     self.subscriber_to_hostport_map.pop(self)
 
 
+class TestHandler(tornado.web.RequestHandler):
```

Comment:
I thought it might be unnecessary to nest this within `WebGuiServer`. But I know we already nested `MainHandler` and `PlugsHandler`, so let me know what your preference is.